04.ls/wc.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  options, paths = receive_command_argument
+  paths.empty? ? output(count_stdin, options) : output(calculate_file_text(paths), options)
+end
+
+def receive_command_argument
+  opt_params = { l: false, w: false, c: false }
+  opt = OptionParser.new
+
+  opt.on('-l') { |v| opt_params[:l] = v }
+  opt.on('-w') { |v| opt_params[:w] = v }
+  opt.on('-c') { |v| opt_params[:c] = v }
+  argv = opt.parse(ARGV)
+  [opt_params, argv]

```

Comment:
```suggestion
  [options, paths]
```

8行目の命名に合わせておくとなお良いです。同じデータには同じ名前を付けた方が理解しやすいので。

Reply:
修正いたしました